### ISSUE TITLE
pipeKSampler: varying results on the same workflow

### ISSUE THREAD

**Issue by mizukarada** [TinyTerra/ComfyUI_tinyterraNodes#15]
- Happens on a highresfix workflow (1 pipeLoader, 2 pipeKSamplers)
- I think it's related to #9 
- I think there's also a bug when you set the first pipeKSampler to `hold` along with your seed set to random, the 2nd pipeKSampler will use a random latent input.

## Reproduction
1. Use a fixed seed. Generate an image and its upscaled image. On the 2nd pipeKSampler change its latent upscale to something else like `bilinear`. Immediately cancel and change it back to your previous latent upscaler, e.g. `nearest-exact`.
2. The results will be different from your first time generating the upscaled image
3. You can keep repeating this process, constantly going back to `nearest-exact` or your upscaler of your choice, and it will be different each time.

**Comment by TinyTerra** [comment]
Please let me know if 0bba92f0942a781f56374b31e8adc1ca10e36794 fixes both of those issues for you

**Comment by mizukarada** [comment]
Thanks! The issue seems to be fixed now, but I get this error if I use pipeEdit:

```log
!!! Exception during processing !!!
Traceback (most recent call last):
  File "C:\Users\anon\Downloads\Programs\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\anon\Downloads\Programs\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\anon\Downloads\Programs\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\anon\Downloads\Programs\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_tinyterraNodes\tinyterraNodes.py", line 1533, in flush
    new_model, new_pos, new_neg, new_latent, new_vae, new_clip, new_image, new_seed = pipe['orig'].values()
                                                                                      ~~~~^^^^^^^^
KeyError: 'orig'
```

**Comment by TinyTerra** [comment]
Oh of course.. should be fixed in 19fe90db224c0ceaf4ed553a6f6137f700ad2cc0 :)

**Comment by mizukarada** [comment]
Thanks! It's fixed now.
